circle.yml

```diff
@@ -18,6 +18,7 @@ dependencies:
 test:
   override:
     - "cd $REPO && make ci"
+    - "cd $REPO/examples/basecoin && make get_vendor_deps && make build && make test"
```

Comment:
we might want a better way to test all the examples

Reply:
I kind of like this - I want the examples to really be more-less standalone